packages/custom_lint/lib/custom_lint.dart

```diff
@@ -148,20 +166,45 @@ void _renderLints(
   if (errors.isEmpty) {
     stdout.writeln('No issues found!');
     return;
+  } else {
+    for (final error in errors) {
+      stdout.writeln(
+        '  ${_relativeFilePath(error.location.file, workingDirectory)}:${error.location.startLine}:${error.location.startColumn}'
+        ' • ${error.message} • ${error.code}',
+      );
+    }
   }
 
-  exitCode = 1;
+  var hasErrors = false;
+  var hasWarnings = false;
+  var hasInfos = false;
   for (final error in errors) {
-    stdout.writeln(
-      '  ${_relativeFilePath(error.location.file, workingDirectory)}:${error.location.startLine}:${error.location.startColumn}'
-      ' • ${error.message} • ${error.code}',
-    );
+    hasErrors |= error.severity == AnalysisErrorSeverity.ERROR;
```

Comment:
Don't use `|=`. This is the `|` operator, not `||`. 
So the == would be executed even though it's unnecessary.

Reply:
fixed commit 36140d7